server/login.js

```diff
@@ -52,16 +55,20 @@ setTimeout(function() {
  *
  * Type: WS
  */
-exports.logoutUser = function (link) {
-    var session = link.session;
+exports.logoutUser = function (stream) {
+    stream.data(function (err, data) {
+        var session = stream.socket.session;
 
-    if (!session || !session.destroy) {
-        return link.end('You are already out. Stay out!');
-    }
+        if (!session || !session.destroy) {
+            stream.write('You are already out. Stay out!');
+            return stream.end();
+        }
 
-    // destroy session
-    session.destroy(function () {
-        link.end();
+        // destroy session
+        session.destroy(function () {
+            stream.write();
```

Comment:
Yes. Isn't `stream.end()` enough? Or what does `stream.write()` do in this case?